portlets/src/main/webapp/vue-app/connector-admin-settings/components/AdminConnectorSettings.vue

```diff
@@ -106,15 +106,15 @@ export default {
           const filteredList = this.adminConnectorsExtensions.filter(connectorExtension => !connectors.some(item => item.name === connectorExtension.componentOptions.name)).map(item => ({
             name: item?.componentOptions?.name,
             title: item?.componentOptions?.title,
+            comingSoon: item?.componentOptions?.comingSoon,
```

Comment:
May be passing all options all time, WDYT ?

```suggestion
            .map(item => item?.componentOptions).filter(e => e)
```

Reply:
Done!